tabooScraper.py

```diff
@@ -0,0 +1,41 @@
+from urllib.request import urlopen
+import re
+
+def parseOutput(url):
+    '''
+    Examples Input/Output
+    Input: https://playtaboo.com/ajax/v1/next/?1624643694954
+    Output: (GOVERNOR, ['STATE', 'REPRESENTATIVE', 'LEADER', 'EACH', 'ELECT'])
+    '''
+    string = str(urlopen(url).read())
+
+    gameWordQuery = r'game-word\\\\">'+'(.*?)'+r'<\\\\/h2>'
```

Comment:
these new queries seem complicated, not sure if there is an easier to achieve the same

Reply:
but it works 👍🏾

Reply:
so, `r'...'` is the raw text found in the string of the web page. In other words, the query is really `r'____' + '(.*?)' + r'___'`. But I could add a comment to make this info clear